### PR TITLE
Introducing Inventory Changer

### LINKED ISSUE #487: Inventory Changer
I would love to see the inventory changer in this cheat, it's so cool.

### LINKED ISSUE #1379: [idea] inventory changer
i've seen most csgo cheats nowadays have a inventory changer and idk how to do it myself so i would like this to happen

### LINKED ISSUE #1516: Inventory changer (request)
Hello! It's possible to add inventory changer? example: https://www.youtube.com/watch?v=GXYrp8IBWPA&t=324s

### LINKED ISSUE #1590: [Petition] Inventory Changer
What if you replaced skin changer with an inventory changer? That would be better (and pretty sick) in my opinion. If possible can you do it please?

### LINKED ISSUE #1933: Music changer
that about making not only a skin changer but also a music changer, I think it would be pretty good, since not everyone wants to pay for music and music in cs is in any case more atmospheric than outside.

### LINKED ISSUE #2075: inventory changer?
Does anyone have the code for an inventory changer? I want to put it in Osiris because I personally prefer inventory changer to skin changer.

### LINKED ISSUE #2192: [REQUEST] : Add Inventory changer?
the title says everything.

### LINKED ISSUE #2463: [Suggestion] Inventory changer
we need inventory changer.
theres some code that i found, i dont know how is it going to work, if someone succeeds to import it into Osiris, please respond:
https://www.unknowncheats.me/forum/counterstrike-global-offensive/247189-inventory-changer.html (not ad)

### LINKED ISSUE #2572: Inventory changer
you can add inventory changer and medails and more features

### LINKED ISSUE #2652: Skin changer issue
When I'm trying to switch my knife's skin to a butterfly knife skin, it just switches to a Bayonet skin instead of saving it as a butterfly knife skin, the skins have the same names though, so it just crashes if I don't switch it

### LINKED ISSUE #2768: [suggestion] seperate skins for ct and tside
would be cool (very cool)

### LINKED ISSUE #2819: Weird Issue (Skin changer).
Problem.:
In skin changer tab i have saved m9 bayonet(knife) with m9 bayonet doppler skin(Paint kit) in my config, but when i load my config the "Paint Kit" tab automatically changes to (bayonet : doppler ) rest every thing remains unchanged i mean wear, seed,etc. other weapons remain same. i only face this problem in the knife tab. 

Solution tried: 
1. Changing knife to Other knife example butterfly, knife changes but paint kit remains same in my case bayonet : doppler.
2. config delete and making new one. 
3. Osiris rebuild.
4. Osiris clean and build again. 
5. gaming reinstallation.

Image Link.: https://imgur.com/oiDbB2i

Config Link.: https://anonfiles.com/R1l521s3uc/SHUBU

Osiris build.: https://anonfiles.com/rbl92fs2u8/Osiris_dll

Request.: please help.

### PR DESCRIPTION
**Pull request by danielkrupinski**
Inventory Changer comes with a full support for both **Windows** and **Linux**. All skins are in "Factory New" condition and have random wear and seed. You can change wear and seed by editing the config file.

### Add items to your inventory:
- weapon skins
- knives
- gloves
- stickers
- music kits
- collectible pins
- service medals
- and more...

### Apply sticker
sticker_apply

### Scrape sticker
sticker_scrape

### Name your weapon...
name_tag

### ...or undo it
name_tag_remove

### Issues resolved
- closes #2819
- closes #2768
- closes #2652
- closes #2572
- closes #2463
- closes #2192
- closes #2075
- closes #1933
- closes #1590
- closes #1516
- closes #1379
- closes #487

### CODE REVIEW

**Comment by blackMasterLord** [comment]
### 0_0

**Comment by wemanzoz** [comment]
SHEESH

**Comment by Zvondaaa** [comment]
cool dude

**Review by rollraw — COMMENTED** [review]
i didnt look through all changes, but if u're using inventory changer it should change music kit just on equip, at least for me it works (sorry for review, missclick)

File: Osiris/Hacks/InventoryChanger.cpp [hunk 1]
```diff
@@ -0,0 +1,1728 @@
...
+
+struct DynamicGloveData {
+    float wear = 0.0f;
+    int seed = 1;
+};
+
+static std::vector<DynamicSkinData> dynamicSkinData;
+static std::vector<DynamicGloveData> dynamicGloveData;
+
+constexpr auto BASE_ITEMID = 1152921504606746975;
```
Comment (rollraw) [inline Osiris/Hacks/InventoryChanger.cpp:148]: like i understand all items you will add will be on top with this method, even if u receive other item from game. there's other way to do it

Comment (danielkrupinski) [reply Osiris/Hacks/InventoryChanger.cpp:148]: This is method guarantees that there will be no collision between real and fake item IDs.

**Comment by caiocinel** [comment]
I was really wondering about the low number of commits in the last few days. That explains everything.

**Comment by r3s1l13nt** [comment]
Lets goooo

**Pull request merged**